source/gameengine/Ketsji/KX_Scene.cpp

```diff
@@ -1637,6 +1637,10 @@ void KX_Scene::LogicEndFrame()
 	//prepare obstacle simulation for new frame
 	if (m_obstacleSimulation)
 		m_obstacleSimulation->UpdateObstacles();
+
+	for (CListValue::iterator<KX_FontObject> it = m_fontlist->GetBegin(), end = m_fontlist->GetEnd(); it != end; ++it) {
```

Comment:
Currently we can't but i investigated on making it possible, i'm going to back to my investigations.